ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -176,8 +189,14 @@ public GetBlobOperationTest(String operationTrackerType) throws Exception {
     MockNetworkClientFactory networkClientFactory =
         new MockNetworkClientFactory(vprops, mockSelectorState, MAX_PORTS_PLAIN_TEXT, MAX_PORTS_SSL,
             CHECKOUT_TIMEOUT_MS, mockServerLayout, time);
+    if (testEncryption) {
+      kms = new MockKeyManagementService(new KMSConfig(vprops),
```

Comment:
If the setup/shutdown of the crypto components is generally the same in all of the router unit tests, this code could be moved into a `CryptoComponents` utility class. The class could also have a shutdown method that will shutdown cryptoJobHandler and kms.

Reply:
as mentioned earlier, will be revamping in next patch.